### ISSUE TITLE
Styling issues in TypeScript LSP hover docs : black background to text

### ISSUE THREAD

**Issue by David-Else** [helix-editor/helix#1948]
### Summary

With the TypeScript language server there is an unwanted background of black on all the text in the hover. The dialogue box is grey, but each character of the text is in a black box:

![Screenshot from 2022-04-03 20-09-15](https://user-images.githubusercontent.com/12832280/161444243-53c83d32-aa3f-47ff-b66c-aeb4fc52d909.png)

### Reproduction Steps

Just look at any typescript docs with `space k`

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

linux

### Terminal Emulator

kitty

### Helix Version

helix 22.03 (d4e45fd4)

**Comment by kirawi** [comment]
I'm guessing it's this line right here that's causing it:
https://github.com/helix-editor/helix/blob/9782204f739813ab5b2dd82451ef2d4695b41d22/runtime/themes/dark_plus.toml#L77

Could you try deleting `bg = "background"`? It's been a while, so I don't remember why I originally wrote that.

**Comment by David-Else** [comment]
@kirawi Great to hear from the theme creator, deleting `bg = "background"` works!

I have been comparing your theme to VS Code online https://vscode.dev/github/microsoft/vscode and Neovim https://github.com/Mofiqul/vscode.nvim, and it is good! :)

I have found a problem that Markdown looks bad, loads of weird colors that are not part of the theme. Can this be fixed in the theme now? I think the Markdown treesitter parser  https://github.com/MDeiml/tree-sitter-markdown/commits/main is semi-new in helix? I only found out about Helix yesterday, can't believe how good it is.

**Comment by kirawi** [comment]
Yeah I didn't get around to adding the markdown colors. I think the current ones are the ones used by the default theme. I'll try to see if I can do that sometime soon.

**Comment by kirawi** [comment]
I updated the colors in https://github.com/helix-editor/helix/pull/1989

**Comment by kirawi** [comment]
I'll look into that too.

**Comment by David-Else** [comment]
@kirawi Also the HTML highlighting seems wrong:

# helix
![Screenshot from 2022-04-07 20-25-56](https://user-images.githubusercontent.com/12832280/162281432-1d85c1aa-eabd-4be7-bf39-40db23f2fe76.png)
# neovim using treesitter https://github.com/Mofiqul/vscode.nvim
![Screenshot from 2022-04-07 20-26-26](https://user-images.githubusercontent.com/12832280/162281436-4717e5cf-2244-42a2-913f-e8e00f395f9f.png)

# vs code online https://vscode.dev/github/microsoft/vscode
![Screenshot from 2022-04-07 20-29-45](https://user-images.githubusercontent.com/12832280/162281727-33fd430f-4422-4d34-87e4-dd7495d6803b.png)

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Img Align Attribute</title>
  </head>
  <body>
    <p>
      This is an example. <img src="image.png" alt="Image" align="middle" /> More text right here
      <img src="image.png" alt="Image" width="100" />
    </p>
  </body>
</html>
```

**Comment by kirawi** [comment]
Both will be fixed in #2014
